brianlong/www.validators.app
# 186685541 score for skipped after

#### What's this PR do?
- add skipped after score to views

#### How should this be manually tested?
- run tests
- run gather and score daemons
- check if skipped_after_score is added correctly to validator_score_v1s
- check if all the validator views include skipped after score
- check api docs and getting started

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186685541)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
